### PR TITLE
add notification service

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**

### CODE REVIEW

File: frontend/src/app/core/services/notification.service.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { inject, Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Injectable({ providedIn: 'root' })
+export class NotificationService {
+    public constructor(private snackBar: MatSnackBar) {
+        this.snackBar = inject(MatSnackBar);
+    }
+
+    public showErrorMessage(error: any) {
```
Comment (tatianahutii) [inline frontend/src/app/core/services/notification.service.ts]: Do not use any

File: frontend/src/app/core/services/notification.service.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { inject, Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Injectable({ providedIn: 'root' })
+export class NotificationService {
+    public constructor(private snackBar: MatSnackBar) {
```
Comment (tatianahutii) [inline frontend/src/app/core/services/notification.service.ts:6]: We need to  have more types of messages
info, warning, error, success

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/styles/material.sass [hunk 3]
```diff
@@ -6,3 +6,18 @@
  *.mat-tab-group.mat-primary .mat-ink-bar
  *    background-color: red
  */
+
+snack-bar-container
+    &.error-snack-bar 
+        background: #a90707
```
Comment (tatianahutii) [inline frontend/src/styles/material.sass]: Please do not use hardcoded colors, use our color variables

File: frontend/src/styles/material.sass [hunk 4]
```diff
@@ -6,3 +6,18 @@
...
+        text-align: center
+    &.success-snack-bar 
+        background: #07be53
```
Comment (tatianahutii) [inline frontend/src/styles/material.sass]: See previous comment

File: frontend/src/styles/material.sass [hunk 5]
```diff
@@ -6,3 +6,18 @@
...
+        text-align: center
+    &.warning-snack-bar 
+        background: #d68408
```
Comment (tatianahutii) [inline frontend/src/styles/material.sass]: See previous comment

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/styles/material.sass (hunk 5 above)
Comment (SlakvichAleksandr) [reply frontend/src/styles/material.sass]: I need to merge new version of development branch, because now we have separate sass file for colors

**Review by tatianahutii — APPROVED** [review]
LGTM

File: frontend/src/app/core/services/notification.service.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        this.snackBar = inject(MatSnackBar);
+    }
+
+    public showErrorMessage(error: string) {
+        this.snackBar.open(error, '', { duration: 2500, panelClass: 'error-snack-bar' });
```
Comment (tatianahutii) [inline frontend/src/app/core/services/notification.service.ts]: Why one of the parameters is an empty string?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/core/services/notification.service.ts (hunk 6 above)
Comment (SlakvichAleksandr) [reply frontend/src/app/core/services/notification.service.ts]: I should make methods with 2 parameters or use some global constant for empty string ?

Comment (SlakvichAleksandr) [reply frontend/src/app/core/services/notification.service.ts]: I should make methods with 2 parameters or just use some kind of global constant for empty string ?

Comment (SlakvichAleksandr) [reply frontend/src/app/core/services/notification.service.ts]: We have notification without button. In snackBar.open second parameter it's name for action. That's why empty string is used

**Review by tatianahutii — APPROVED** [review]
LGTM

**Pull request merged**